pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/dto/dashboard/DashboardDto.java

```diff
@@ -0,0 +1,84 @@
+package org.broadinstitute.dsm.db.dto.dashboard;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import lombok.Data;
+import org.broadinstitute.dsm.model.dashboard.DisplayType;
+import org.broadinstitute.dsm.model.dashboard.Size;
+
+@Data
```

Comment:
Please add a comment explaining what a "Dashboard" is intended to do, and what sorts of data it should contain